lib/command/base.rb

```diff
@@ -517,6 +550,36 @@ def ensure_docker_running!
       raise "Can't run Docker. Please make sure that it's installed and started, then try again."
     end
 
+    def run_hook(script_or_command, title:)
+      if potential_file?(script_or_command)
```

Comment:
This is not right.

We should **ASSUME** that anything referenced is either relative or in the path of execution, so there is no difference between a command with zero args and a command with args.